backend/EasySpeak.Core/EasySpeak.Core.BLL/Services/UserService.cs

```diff
@@ -87,5 +90,14 @@ public async Task<string> UploadProfilePhoto(IFormFile file)
 
             return profilePhoto.Url;
         }
+
+        private async Task<string> GetProfileImageUrl(long? imageId)
+        {
+            var profileImage = await _context.EasySpeakFiles.FirstOrDefaultAsync(f => f.Id == imageId);
+            
+            var imageUrl = profileImage!.Url;
+
+            return imageUrl!;
```

Comment:
Maybe we can not create variable, but immediately return the result?

Reply:
Fix it